core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -41,6 +41,7 @@ describe('Topic editor functionality', function() {
   var skillEditorPage = null;
   var explorationEditorPage = null;
   var explorationEditorMainTab = null;
+  var UNASSIGNED = 'Unassigned';
```

Comment:
Please recall the usual conventions for defining named constants: this should be SKILL_STATUS_UNASSIGNED, here and in the other 5 places.

Reply:
Fixed.